tensorflow/compiler/xla/tests/exhaustive/exhaustive_op_test_utils.h

```diff
@@ -943,6 +952,23 @@ inline std::vector<std::pair<int64_t, int64_t>> CreateExhaustiveF32Ranges() {
   return result;
 }
 
+template <typename T>
+bool CheckValidRange(const T input, const T max, const T min) const {
+  return (input < max) && (input > min);
+}
+
+template <>
+bool CheckValidRange(const std::complex<double>, const std::complex<double>,
+                     const std::complex<double>) const {
```

Comment:
non-member function cannot have 'const' qualifier

Reply:
How are these not caught by the Github CI btw?